web/test/pages/profile/profile-shared.test.tsx

```diff
@@ -439,13 +439,16 @@ describe("profile - shared", () => {
           }),
         });
         renderPage({ business, isAuthenticated: IsAuthenticated.FALSE });
-        expect(
-          screen.getByText(Config.profileDefaults.default.noteForBusinessesFormedOutsideNavigator),
-        ).toBeInTheDocument();
+        const calloutText =
+          Config.profileDefaults.default.noteForBusinessesFormedOutsideNavigator.match(
+            /headerText="([^"]*)"/,
+          )?.[1];
+        expect(calloutText).toBeDefined();
+        expect(screen.getByText(calloutText!)).toBeInTheDocument();
```

Comment:
`calloutText` should always be defined, but covering my bases since the next line includes a non-null assertion